agent/engine/metadataservice/write_metadata.go

```diff
@@ -0,0 +1,88 @@
+package metadataservice
+
+import (
+	"io/ioutil"
+	"os"
+	"regexp"
+
+	"github.com/aws/amazon-ecs-agent/agent/api"
+//	"github.com/cihub/seelog"
+	//docker "github.com/fsouza/go-dockerclient"
+)
+
+const (
+	taskArnRegexExpr = "(:task/)|(arn:aws:ecs:)|:"
```

Comment:
The main reason I did it this way is to also remove the "task/" part of the task-id. This can be dealt with by a second strings.SplitN but I felt it was more clean to use a Regex split instead.